.github/ISSUE_TEMPLATE/rendering-bug-report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Rendering bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**LaTeX code**
+The code of LaTeX you tried to render:
+```latex
+
+```
+
+**Expected behavior**
+A clear and concise description of what you expected to happen. You can obtain a LaTeX rendering using [QuickLaTeX](https://www.quicklatex.com/).
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Environment (please complete the following information):**
+ - Device: [e.g. Desktop, iPhone 8]
+ - OS: [e.g. Windows 10, iOS 11]
+ - Browser: [e.g. Chrome, Safari]
+ - Version: [e.g. 72]
+
+**Additional context**
```

Comment:
Minor style thing: It'd be nice to end the bold headings with colon or period (maybe currently have no punctuation). I think this will make them easier to read when rendered (and the newlines disappear).